sspl_test/alerts/node/test_node_bmc_interface.py

```diff
@@ -14,100 +14,87 @@
 # cortx-questions@seagate.com.
 
 # -*- coding: utf-8 -*-
-import time
 import subprocess
 
-from default import world
-from messaging.ingress_processor_tests import IngressProcessorTests
-from messaging.egress_processor_tests import EgressProcessorTests
-from common import check_sspl_ll_is_running
 from framework.base.sspl_constants import DATA_PATH
 from alerts.node import simulate_bmc_interface_alert
+from framework.base.testcase_base import TestCaseBase
 
 
-def init(args):
-    pass
-
-def test_bmc_interface(args):
-    check_sspl_ll_is_running()
-    # backup active bmc interface
-    BMC_IF_CONSUL_KEY,BMC_IF_CONSUL_VAL = backup_bmc_config()
-
-    if BMC_IF_CONSUL_VAL == "lan":
-        simulate_bmc_interface_alert.lan_channel_alert(BMC_IF_CONSUL_KEY,BMC_IF_CONSUL_VAL)
-    else:
-        simulate_bmc_interface_alert.kcs_channel_alert(BMC_IF_CONSUL_KEY,BMC_IF_CONSUL_VAL)
-
-    bmc_interface_message = None
-    for i in range(30):
-        if world.sspl_modules[IngressProcessorTests.name()]._is_my_msgQ_empty():
-            time.sleep(2)
-        while not world.sspl_modules[IngressProcessorTests.name()]._is_my_msgQ_empty():
-            ingressMsg = world.sspl_modules[IngressProcessorTests.name()]._read_my_msgQ()
-            print("Received: %s" % ingressMsg)
-            try:
-                # Make sure we get back the message type that matches the request
-                msg_type = ingressMsg.get("sensor_response_type")
-                if msg_type["info"]["resource_type"] == "node:bmc:interface:kcs" or \
+class BmcInterfaceTest(TestCaseBase):
```

Comment:
What extra we are achieving here functionally ? Putting existing code under a class, derived from a Parent to exhibit common behavior, is fine.